build/source/org/zotero/integration/ooo/comp/Document.java

```diff
@@ -248,13 +248,21 @@ public boolean importDocument() throws Exception {
 				XEnumerationAccess.class, text);
 		XEnumeration xParaEnum = xParaAccess.createEnumeration();
 		ArrayList<XTextContent> removeParagraphs = new ArrayList<XTextContent>();
-		for (int i = 0 ; i < 3 && xParaEnum.hasMoreElements(); i++) {
+		for (int i = 0 ; i < 4 && xParaEnum.hasMoreElements(); i++) {
```

Comment:
This line shouldn't change.

Reply:
But the export creates 4 new paragraphs, not 3.

Please see the test documents where the first line is marked "MY FIRST LINE".

This should be the first line after importing too!

Reply:
Ok, I just checked the other plugins to make sure this won't cause issues, and yes, all others remove 4, so this is fine.

Reply:
you drive me crazy

Reply:
Sorry, but the general rule is don't fix what's not broken. I spent a ton of time on making the import/export code cross-compatible between word processors, so any changes have to be warranted and require cross-testing and checking.

Reply:
> Sorry, but the general rule is don't fix what's not broken. I spent a ton of time on making the import/export code cross-compatible between word processors, so any changes have to be warranted and require cross-testing and checking.

I'm kidding! I know the difficulties in cross-compatibility! ;)